content.properties

```diff
@@ -0,0 +1,25 @@
+name=${project.artifactId}
+version=${project.version}
+
+var.patientIdentifierType.nif.uuid=e797f826-8e8f-11e7-bb31-be2e44b06b34
+var.patientIdentifierType.cin.uuid=e797face-8e8f-11e7-bb31-be2e44b06b34
+var.patientIdentifierType.biometricsReferenceCode.uuid=e26ca279-8f57-44a5-9ed8-8cc16e90e559
+var.patientIdentifierType.zlEmrId.uuid=a541af1e-105c-40bf-b345-ba1fd6a59b85
+var.patientIdentifierType.dossierNumber.uuid=e66645eb-03a8-4991-b4ce-e87318e37566
+var.patientIdentifierType.externalDossierNumber.uuid=9dbea4d4-35a9-4793-959e-952f2a9f5347
+var.patientIdentifierType.hivemrV1.uuid=139766e8-15f5-102d-96e4-000c29c2a5d7
+var.patientIdentifierType.userEnteredRef.uuid=d9009bd0-eac9-11e5-a837-0800200c9a66
+var.patientIdentifierType.dentalDossierNumber.uuid=b5d0a5bd-adf3-4fe0-a231-5a488f6d2c61
+var.patientIdentifierType.hivDossierNumber.uuid=3B954DB1-0D41-498E-A3F9-1E20CCC47323
+var.patientIdentifierType.hivemrV1InfantId.uuid=B68489FE-7B34-400B-992C-D4C1B5DD2928
+var.patientIdentifierType.hivemrV1InfantCode.uuid=5FB435B4-F573-48C0-A715-FEBD42FDBB75
+
+var.encounterType.PATIENT_REGISTRATION.name=Enregistrement de patient
+var.encounterType.CHECK_IN.name=Inscription
+var.encounterType.MEDICATION_DISPENSED.name=Médicaments administrés
+var.encounterType.VITALS.name=Signes vitaux
+
+var.concept.prenatalGroup.uuid=f58e875f-de07-4cfd-8fc7-3e70b7390ecc
+var.concept.prenatalIndividual.uuid=d16fc2d4-e736-4372-8cbf-99542e5dac5e
+var.concept.pedsGroup.uuid=d087f783-de30-4aea-8a4d-50ec72ed656a
+var.concept.pedsIndividual.uuid=59f75bbf-4a6f-42b9-92d4-9259cba548d2
```

Comment:
Looks like you went with variable inheritance in the end (since I'm not seeing the program uuids here?)

Reply:
No, not really.  See the pom.